pkg/project/project_test.go

```diff
@@ -16,6 +16,80 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+func TestCreate(t *testing.T) {
```

Comment:
Shouldn't this be a unit test for `client .CreateNewProject` rather than  `project.Create`.
Most logic is in   `client .CreateNewProject`and this test looks like it testing that, rather what is in  `project.Create`.

Reply:
> Shouldn't this be a unit test for `client .CreateNewProject` rather than `project.Create`.
> Most logic is in `client .CreateNewProject`and this test looks like it testing that, rather what is in `project.Create`.

I will add one more test for `client .CreateNewProject` function.

Reply:
> > Shouldn't this be a unit test for `client .CreateNewProject` rather than `project.Create`.
> > Most logic is in `client .CreateNewProject`and this test looks like it testing that, rather what is in `project.Create`.
> 
> I will add one more test for `client .CreateNewProject` function.

@prietyc123 You can create a separate pr for ```client .CreateNewProject``` UTs. @mik-dass can you please do one more round of review and add the label if looks good.

> Shouldn't this be a unit test for client .CreateNewProject rather than project.Create.
Most logic is in client .CreateNewProjectand this test looks like it testing that, rather what is in project.Create.

@mik-dass Can you please respond to this